packages/components/src/presentation-components/tree/FilteredDataProvider.ts

```diff
@@ -142,6 +142,7 @@ export class FilteredPresentationTreeDataProvider implements IFilteredPresentati
   }
 
   public getNodeKey(node: TreeNodeItem): NodeKey {
+    // eslint-disable-next-line deprecation/deprecation
```

Comment:
why not switch to recommended way of doing this?

Reply:
This method is deprecated and will require that invalid key is returned if supplied node is not `PresentationTreeNodeItem` so I wanted to reuse implementation from `PresentationTreeDataProvider` which takes care of creating invalid `NodeKey`